<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqLogViewerWindow</class>
 <widget class="QMainWindow" name="pqLogViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="margin">
     <number>4</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="settingsLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QWidget" name="newLogWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="newLogLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>New Log</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="newLogFrame">
           <property name="frameShape">
            <enum>QFrame::HLine</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="processLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Process</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="processComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Application process to log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rankLabel">
        <property name="text">
         <string>Rank</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="rankComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Rank of process to log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addLogButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add Log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="globalFilterSettings" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="globalFilterLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Global Filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="globalFilterFrame">
           <property name="frameShape">
            <enum>QFrame::HLine</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="globalFilter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Global filter that applies to all logs</string>
        </property>
        <property name="placeholderText">
         <string>GlobalFilter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="verbositySettings" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Verbosity Settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::HLine</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="verbosityLevelLabel">
        <property name="text">
         <string>Client Verbosity Level</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="clientVerbosities"/>
      </item>
      <item>
       <widget class="QLabel" name="serverVerbosityLabel">
        <property name="text">
         <string>Server Verbosity Level</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="serverVerbosities"/>
      </item>
      <item>
       <widget class="QLabel" name="dataServerVerbosityLabel">
        <property name="text">
         <string>Data Server Verbosity Level</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="dataServerVerbosities"/>
      </item>
      <item>
       <widget class="QLabel" name="renderServerVerbosityLabel">
        <property name="text">
         <string>Render Server Verbosity Level</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="renderServerVerbosities"/>
      </item>
      <item>
       <widget class="QWidget" name="categoryElevationWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>8</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="categoryElevationVerbosity">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Category Verbosity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="categoryElevationFrame">
           <property name="frameShape">
            <enum>QFrame::HLine</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="categoryLabel">
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="categoryComboBox">
        <property name="toolTip">
         <string>Select a log category whose verbosity you want to change</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="targetLabel">
        <property name="text">
         <string>Verbosity Level</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="targetVerbosities">
        <property name="toolTip">
         <string>Set desired verbosity of log category on all processes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetAllCategoryVerbositiesButton">
        <property name="text">
         <string>Reset All Category Verbosities</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearLogsButton">
        <property name="text">
         <string>Clear Logs</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="logTabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
